src/components/presentational/Sidebar/Sidebar.js

```diff
@@ -63,7 +63,11 @@ export default class Sidebar extends PureComponent {
       const scrollPageTop = Math.max(document.documentElement.scrollTop, document.body.scrollTop);
       let sidebarTop = headerHeight - scrollPageTop;
 
-      sidebarTop = sidebarTop > 0 ? sidebarTop : 0;
+      if (0 === scrollPageTop) {
```

Comment:
@BogdanScherban `scrollPageTop === 0` would be more readable

Reply:
OK, I'll change it